samples/java/junit5-selenium/src/test/java/com/idera/testrail/tests/HomePageTest.java

```diff
@@ -0,0 +1,78 @@
+package com.idera.testrail.tests;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+class HomePageTest {
+
+    public static WebDriver driver;
+
+    @BeforeEach
+    public void setUp() {
+        String os = SystemUtils.OS_NAME;
+        String pathToChromeDriver = null;
+        if (os.contains("Windows")) {
+            pathToChromeDriver = "drivers/windows/chromedriver.exe";
+        } else {
+            pathToChromeDriver = "drivers/linux/chromedriver";
+        }
```

Comment:
maybe using webdrivermanager would make this more clean and less prone to actual details of setting the chromedriver? 

https://github.com/bonigarcia/webdrivermanager

Reply:
@bitcoder as discussed , I would  not like to change it . The reason is ,  in CircleCI we are using the image with browser preinstalled on  it and we can also define the particular version of the browser which we would like to install . So in our CircleCI.yaml file , we already fixed chrome version 109 and I have placed appropriate drivers in the drivers folder

Reply:
@d-rede its removed now